metadata/sandbox.go

```diff
@@ -239,6 +239,9 @@ func (s *sandboxStore) Delete(ctx context.Context, id string) error {
 		}
 
 		if err := buckets.DeleteBucket([]byte(id)); err != nil {
+			if err == bbolt.ErrBucketNotFound {
+				err = errdefs.ErrNotFound
+			}
```

Comment:
This looks like maybe it should be in a different PR?

Reply:
Clearly I didn't read the PR description.  But I do think it's worth separating "pkg/cri: should ignore no sandbox bucket" into a different PR so we can merge it more quickly.

Reply:
Yes. The main branch can't delete pod which created by v1.6 release, since there is no sandbox record or bucket for them. I will send the individual pull request for this.